NodeBase/Dockerfile

```diff
@@ -13,6 +13,7 @@ USER root
 RUN apt-get update -qqy \
   && apt-get -qqy install \
     xvfb \
+    pulseaudio \
```

Comment:
I can see that the video does not load in Firefox, but it does load in Chrome. Having said that, why is `pulseaudio` installed for both? Shouldn't it be only for Firefox?

Reply:
The video play is fixed by installing the `libavcodec-extra` package here, only for Firefox. https://github.com/SeleniumHQ/docker-selenium/pull/994/files#diff-a7327fbb68be844da4c26c10bc15c5c2R16

The installation of `pulseaudio` package for all browsers is because Chrome is also affected by the memory leak caused if there is no sound consumption.

You can check it. Simply load an HLS video site in either browser and wait. You could see a continuous increase in memory consumption until the OOM kill of the container.

Reply:
Let's revert this change also, since we don't modify these Dockerfile's directly :+1:

Reply:
@ddavison Done.